README.md

```diff
@@ -17,12 +17,21 @@ $ npm install --save inquirer-npm-name
 var inquirer = require('inquirer');
 var askName = require('inquirer-npm-name');
 
+//The legacy approach: Using callback
 askName({
   name: 'name',
   message: 'Module Name'
 }, inquirer, function (name) {
   console.log(name);
 });
+
+//The latest approach: Using promises
```

Comment:
Yeah, yeah...you are sure. I think in put a git `tag` version reference with this legacy form. But, I'm agree with you :)